### ISSUE TITLE
RFC: Multiple deposits at once

### ISSUE THREAD

**Issue by drjova** [inveniosoftware/invenio-records-js#32]
On CDS we have the use case in which we have in one deposit page multiple deposits. The main goal is to be able to initialise multiple times the ```invenio-records-js``` and each instance to have its own actions. On top of that we will need a way to synchronise all these deposits together.

The proposal is to create the ```invenio-deposits``` which includes multiple deposits.

```html

<invenio-deposits>
 
  <invenio-deposits-alerts />
  <invenio-deposits-actios />

  <invenio-deposit>
     <cds-status />
     <invenio-actions />
     <invenio-form />
  </invenio-deposit>

  <invenio-deposit>
     <cds-status />
     <invenio-actions />
     <invenio-form />
  </invenio-deposit>

</invenio-deposits>
```

**Comment by lnielsen** [comment]
Will this mean big changes for existing forms?

**Comment by drjova** [comment]
Yes, for the time being we have a WIP prototype on CDS we can discuss if we need it for Invenio or not.
